test/helpers/quictestlib.c

```diff
@@ -282,9 +289,14 @@ int qtest_create_quic_objects(OSSL_LIB_CTX *libctx, SSL_CTX *clientctx,
     if (serverctx != NULL && !TEST_true(SSL_CTX_up_ref(serverctx)))
         goto err;
     tserver_args.ctx = serverctx;
+    if (fake_now_lock == NULL)
+        fake_now_lock = CRYPTO_THREAD_lock_new();
```

Comment:
What if this fails? Subsequently we will then call CRYPTO_THREAD_write_lock(NULL)

Reply:
Updated to fail qtest_create_quic_objects  if the lock creation falis